### PR TITLE
User can see other members and how much $ they have

### PR DESCRIPTION
**Pull request by euglazer**
funder-page

bad quality gif, but ya

### CODE REVIEW

**Comment by ahdinosaur** [comment]
#killingit

**Comment by derekrazo** [comment]
This is so great!!! Awesome stuff. Will pull down and check it out later
on. Just got to france and waiting at bus station.

Keep checking the invision as I am pushing lots of updates and things are
changing quite fast. After what's up there now is just incremental little
tweaks though to the projects page.

Full flows for updating project page status coming soon.

File: app/records-interfaces/allocation-records-interface.coffee [hunk 1]
```diff
@@ -0,0 +1,13 @@
+null 
+
+### @ngInject ###
+global.cobudgetApp.factory 'AllocationRecordsInterface', (config, BaseRecordsInterface, AllocationModel) -> 
+  class AllocationRecordsInterface extends BaseRecordsInterface
+    model: AllocationModel
+    constructor: (recordStore) ->
+      @baseConstructor recordStore
+      @restfulClient.apiPrefix = config.apiPrefix
```
Comment (ahdinosaur) [inline app/records-interfaces/allocation-records-interface.coffee]: i think you might need to only do this once in https://github.com/cobudget/cobudget-ui/blob/beta/app/angular-record-store.coffee.

Comment (ahdinosaur) [inline app/records-interfaces/allocation-records-interface.coffee]: nevermind, [as every `restfulClient` is a new instance](https://github.com/loomio/angular_record_store/blob/master/base_records_interface.coffee#L13).

**Pull request merged**